### ISSUE TITLE
 Could not parse the trace log from Chrome TypeError: Cannot read property 'ts' of undefined

### ISSUE THREAD

**Issue by soulgalore** [sitespeedio/browsertime#801]
Since Chrome 73 these errors happens more often than before (seems to happen on our tests in Cnet):

```
[2019-03-20 19:28:12] ERROR: Could not parse the trace log from Chrome TypeError: Cannot read property 'ts' of undefined
    at Object.processEndEvent (/usr/src/app/node_modules/chrome-trace/lib/eventProcessors/durationEvents/completeEventSyntesizer.js:27:46)
    at Object.processEvent (/usr/src/app/node_modules/chrome-trace/lib/eventProcessors/durationEvents/durationProcessor.js:68:35)
    at processors.map.processor (/usr/src/app/node_modules/chrome-trace/index.js:23:45)
    at Array.map (<anonymous>)
    at parseObject (/usr/src/app/node_modules/chrome-trace/index.js:23:18)
    at parseCpuTrace (/usr/src/app/node_modules/browsertime/lib/chrome/cpu.js:9:27)
    at ChromeDelegate.onCollect (/usr/src/app/node_modules/browsertime/lib/chrome/webdriver/chromeDelegate.js:126:25)
    at process._tickCallback (internal/process/next_tick.js:68:7)
```

**Comment by soulgalore** [comment]
Will be fixed in https://github.com/sitespeedio/browsertime/pull/816

**Comment by soulgalore** [comment]
Released in 5.0
